styles.css

```diff
@@ -92,3 +119,239 @@ body {
   padding: 7px;
   height: 15px;
 }
+
+.middle-section-container {
+  display: grid;
+  width: 350px;
+  height: 650px;
+  padding: 15px;
+  border-radius: 20px;
+  border: 2px solid white;
+  align-items: center;
+  margin-bottom: 100px;
+  background-color: white;
+}
+
+.middle-section-container2 {
+  display: grid;
+  width: 350px;
+  height: 700px;
+  padding: 15px;
+  border-radius: 20px;
+  border: 2px solid white;
+  align-items: center;
+  margin-bottom: 100px;
+  background-color: white;
+}
+
+.image-area > img {
+  width: 348px;
+}
+
+.tonic > h1 {
+  font-size: 30px;
+  line-height: 44px;
+  color: #172b4d;
+  font-style: normal;
+  width: 295px;
+  height: 10px;
+}
+
+.tonic2 h1 {
+  font-size: 30px;
+  line-height: 44px;
+  color: #172b4d;
+  font-style: normal;
+  width: 200px;
+}
+
+.canopy-wrapper > ul {
+  display: grid;
+  grid-template-columns: auto auto;
+  color: #7a869a;
+  gap: 20px;
+}
+
+.canopy-wrapper {
+  display: flex;
+  height: 40px;
+}
+
+.canopy-wrapper > p {
+  width: 50px;
+}
+
+.middle-section-nav-bars {
+  display: flex;
+  align-items: flex-start;
+  width: 30px;
+}
+
+.middle-section-nav-bars > ul {
+  display: flex;
+  justify-content: space-evenly;
+  gap: 62px;
+  list-style-type: none;
+  padding-left: 14px;
+  margin: 10px;
+  width: 220px;
+}
+
+.project-link {
+  border: 1px solid blue;
+  width: 95px;
+  height: 35px;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  border-radius: 12px;
+  background-color: #fff;
+  color: #396df2;
+  font-size: 16px;
+}
```

Comment:
- [ ] [OPTIONAL] Please add padding to these buttons as shown in the design. The button should look like the design.